configure.ac

```diff
@@ -1406,7 +1410,15 @@ PKG_CHECK_MODULES(libsystemd, libsystemd >= ${LIBSYSTEMD_MIN_VERSION},
 dnl ***************************************************************************
 dnl python checks
 dnl ***************************************************************************
-if test "x$enable_python" != "xno"; then
+
+if test "x$enable_python2_and_3" = "xyes"; then
+  PKG_CHECK_MODULES(PYTHON3, python3,, AC_MSG_ERROR([Could not find Python 3 development libraries]))
+  PKG_CHECK_MODULES(PYTHON2, python2,, AC_MSG_ERROR([Could not find Python 2 development libraries]))
```

Comment:
The auto detection checks for `python3`, `python2` and `python` in that order, as sometimes there is only `python` (which is actually `python2`) and does not exists `python2` thus this fails.
I would consider checking `python` if the `python2` is not found.
Well I question remains that does `python` always means `python2` or not ?! (sorry I do not know the answer)

Reply:
Sure, I'll add the `python` check.